scripts/docgen/generate-docs.js

```diff
@@ -21,13 +21,16 @@ const fs = require('mz/fs');
 const path = require('path');
 const yaml = require('js-yaml');
 
+// API to generate docs for, 'node' or 'js' - default to 'js'
+const apiType = argv.api || 'js';
```

Comment:
I know it's a very simple API. It would be nice if we can show the option usage with `-help` command.

Reply:
Done.